src/components/Notifications.tsx

```diff
@@ -0,0 +1,55 @@
+const Notifications = Windows.UI.Notifications;
+const ToastTemplateType = Notifications.ToastTemplateType;
+const ToastNotificationManager = Notifications.ToastNotificationManager;
+const ToastNotification = Notifications.ToastNotification;
+
+export function showNotification(notification) {
+  var type = ToastTemplateType.toastText01;
```

Comment:
```suggestion
  let type = ToastTemplateType.toastText01;
```
I'm surprised we don't have lint rules setup about `var`. As a default in our JS, we should always prefer `let`. It's scoped in a way that is much more sensible.